routes/events.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const router = express.Router();
+const eventManager = require('./eventsManager');
+
+router.get('/', (req, res) => {
+  eventManager.getAll()
+  .then(
+    events => res.json(events)
+  ).catch(
+    error => res.status(500).send('Se encontro un error ' + error)
+  )
+})
+
+router.get('/:id', (req, res) => {
+  const idBuscado = req.params.id;
+  eventManager.getEventById(idBuscado)
+  .then(
+    event => {
+      if (event == undefined){
+        res.status(404).send('El evento no existe.');
+      }
+      else {
+        res.json(event);
+      }
+    }
+  ).catch(
+    error => res.status(500).send('Se encontró un error ' + error)
+  )
+})
+
+router.get('/title/:title', (req,res) => {
+  const titleBuscado = req.params.title;
+  eventManager.getEventsByTitle(titleBuscado)
+  .then(
+    events => res.json(events)
+  ).catch(
+    error => res.status(500).send('Se encontró un error ' + error)
+  )
+});
+
+router.post('/', (req,res) => {
+  eventManager.createEvent(req.body.id, req.body.title, req.body.description, req.body.date)
```

Comment:
Thank you, we'll test the fix and commit it to this PR.  How could we prevent the insertion of duplicate events?  I was thinking of a unique index on title + date.